datasets/sipecam.yaml

```diff
@@ -0,0 +1,21 @@
+Name: SiPeCaM (Sitios Permanentes de la Calibración y Monitoreo de la Biodiversidad)
+Description: The SiPeCaM goal is to create a data source that allows to evaluate changes in the biodiversity state, considering key aspect of how does the ecosystem behaves.
+Documentation: https://github.com/CONABIO/sipecam-open-data/blob/main/docs/SiPeCaM.md
+Contact: "Michael F. Schmidt [mschmidt@conabio.gob.mx](mschmidt@conabio.gob.mx)"
+ManagedBy: "[Conabio](https://monitoreo.conabio.gob.mx/)"
+UpdateFrequency: SiPeCaM offers data releases once the biologist recollect the data from the protected natural areas (every 3 months)
+Tags:
```

Comment:
Please stick to tags already in https://github.com/awslabs/open-data-registry/blob/main/tags.yaml. If you think that new tags should be added, please make the changes in tags.yaml and include them in this pull request.